### ISSUE TITLE
Would this adapter work?

### ISSUE THREAD

**Issue by tinkerer9** [sfeakes/AqualinkD#307]
Hello World!

I was wondering if [this adapter](https://www.amazon.com/gp/product/B0BNZBNPWL/ref=ox_sc_act_title_2?smid=A3B0XDFTVR980O&th=1) would work. It says it has the genuine FT232RL chip, but I wasn't sure if it would work.

Thanks a lot!

**Comment by sfeakes** [comment]
Yes, I use that exact adapter for my dev / prototype environment.  It's not as good as the FTDI branded one, but works perfectly fine.
